paper-typeahead.js

```diff
@@ -24,7 +24,7 @@
             }
 
             return data.filter(function(v) {
-              return v.match(r);
+			  return (r.test(v) ? v : null);
```

Comment:
awesome @RNCRyan!

It looks like you are using tabs here, would you mind switching to spaces? thx :sparkles: